cmd/prepare_worker.go

```diff
@@ -120,31 +120,13 @@ func main() {
 }
 
 func (b *workerBuilder) buildPython(ctx context.Context, baseDir string) (sdkbuild.Program, error) {
-	// If version not provided, try to read it from pyproject.toml
-	version := b.version
-	if version == "" {
-		b, err := os.ReadFile(filepath.Join(baseDir, "pyproject.toml"))
-		if err != nil {
-			return nil, fmt.Errorf("failed reading pyproject.toml: %w", err)
-		}
-		for _, line := range strings.Split(string(b), "\n") {
-			line = strings.TrimSpace(line)
-			if strings.HasPrefix(line, "temporalio = ") {
-				version = line[strings.Index(line, `"`)+1 : strings.LastIndex(line, `"`)]
-				break
-			}
-		}
-		if version == "" {
-			return nil, fmt.Errorf("version not found in pyproject.toml")
-		}
```

Comment:
This was a hack and I would like it not to be necessary.

Reply:
Don't all langs support `omes` and `features` runs without specifying an SDK version?

Reply:
OK, I think that to get the "not specifying an SDK version" functionality we should modify `sdkbuild` to allow the version to be missing, and delegate to the package resolvers.

However for now I've got rid of the hacky TOML parsing here and replaced it with a call to `uv tree`.

Reply:
Works for me, though I think `features` repo will need the same treatment at some point